CMakeLists.txt

```diff
@@ -167,6 +167,18 @@ else()
     MESSAGE(FATAL_ERROR "Unknown build type '${CMAKE_BUILD_TYPE}'; valid types are 'Release' or 'Debug'")
 endif()
 
+# Sync with ShadowBuildCommon::cc_flags
```

Comment:
this comment says to sync with `cc_flags`, but you removed the flags in `cc_build` so they're not in sync.

Reply:
Yup, initially I thought we'd need to sync these, but they're already propagated from here to the cc_builds automatically.